fdk/constants.py

```diff
@@ -29,3 +31,9 @@
 CONTENT_LENGTH = "Content-Length"
 FN_ENFORCED_RESPONSE_CODES = [200, 502, 504]
 FN_DEFAULT_RESPONSE_CODE = 200
+
+
+# todo: python 3.8 is on its way, make more flexible
```

Comment:
yea, would be nice to get rid of it - according to pr, issue is just it's a bit slower? i don't think we need the start_serving thing we tried out

Reply:
i see, used for importlib too

Reply:
mind just making this code do `return py_version.major >= 3 && py_version.minor >= 7` ? then we won't have to fuss with it

Reply:
I'll add that when I'd play with 3.8, it's in alpha right now, but it has tons of new features that I'd like to use.